iaso/models/org_unit.py

```diff
@@ -712,10 +713,11 @@ def reject(self, user: User, rejection_comment: str) -> None:
         self.rejection_comment = rejection_comment
         self.save()
 
-    def approve(self, user: User, approved_fields: typing.List[str]) -> None:
+    def approve(self, user: User, approved_fields: typing.List[str], rejection_comment: str = "") -> None:
         self.__apply_changes(user, approved_fields)
         self.updated_by = user
         self.status = self.Statuses.APPROVED
+        self.rejection_comment = rejection_comment
```

Comment:
This is the only thing that bugs me.

`rejection_comment` can now contain rejection or approval comments.

We should rename `rejection_comment` to `comment` because the name of the field is now misleading.

The renaming could be done in another PR (or in this one):

1. add a `comment` field
2. migrate data from `rejection_comment` to `comment`
3. delete `rejection_comment`

@bmonjoie is this easily doable in the mobile app or should we keep `rejection_comment`?

Reply:
Let's keep `rejection_comment` see https://github.com/BLSQ/iaso/pull/1251#issuecomment-2060696621